components/raftstore/src/store/worker/raftlog_fetch.rs

```diff
@@ -42,32 +42,39 @@ impl fmt::Display for Task {
     }
 }
 
-pub struct Runner<EK, ER, R>
+#[derive(Debug)]
+pub struct FetchedLogs {
+    pub context: GetEntriesContext,
+    pub logs: Box<RaftlogFetchResult>,
+}
+
+/// A router for receiving fetched result.
+pub trait LogFetchedNotifier: Send {
```

Comment:
Can we unify `Log` with `Entry` by the way, they are mix-used throughout the code

Reply:
`Log` seems more straightforward. And better do it in other thread.